invokeai/frontend/web/src/features/controlNet/components/parameters/ParamControlNetModel.tsx

```diff
@@ -58,7 +59,7 @@ const ParamControlNetModel = (props: ParamControlNetModelProps) => {
       data.push({
         value: id,
         label: model.model_name,
-        group: MODEL_TYPE_MAP[model.base_model],
+        group: MODEL_TYPE_MAP[model.base_model as BaseModelType],
```

Comment:
These typing issues for controlnet models are resolved in a commit in #4025. We should not cast them to `BaseModelType`. Can you cherry-pick the commit from #4025 please instead?